content/microcontrollers/particle-argon.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Particle Argon"
+weight: 3
+---
+
+[The Particle Argon](https://docs.particle.io/datasheets/wi-fi/argon-datasheet/) is a powerful Wi-Fi enabled development board. It is based on the Nordic [nRF52840](https://www.nordicsemi.com/eng/Products/nRF52840) and ESP32 2.4 GHz Wi-Fi coprocessor.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | YES |
+| I2C      | YES | YES |
+| ADC      | YES | Not yet |
+| PWM      | YES | Not yet |
```

Comment:
It looks like the ADC and PWM is supported? (Also goes for the other two boards)

Reply:
I copied this from nRF52840-MDK. I wasn't sure if ADC and PWM are already implemented for nRF52. I'll try to test and confirm it works this weekend.

Reply:
Any action on this testing @suda ?

Reply:
@deadprogram I did test both ADC and PWM just now and can confirm it works perfectly 👍 Pushed a commit updating this table